snapshots/output/basic-project/lib/more.dart

```diff
@@ -114,17 +112,17 @@
 //    documentation ```dart
 //                 ^^^^ reference scip-dart pub dart:core 2.19.0 dart:core/list.dart/List#
 //                      ^^^ reference scip-dart pub dart:core 2.19.0 dart:core/int.dart/int#
-//                           ^^^^^^^ definition local 1
+//                           ^^^^^^^ definition local 0
```

Comment:
the local variable changes here are just due to no longer declaring `local 0` above, no cause for concern